### ISSUE TITLE
Support removing elements from a circuit

### ISSUE THREAD

**Issue by fb39ca4** [FabriceSalvaire/PySpice#92]
I'd like to be able to isolate parts of a circuit for measuring impedances, etc. by removing elements, and looking through the API documentation for Netlist I can't see a way to do it.

**Comment by FabriceSalvaire** [comment]
**Interesting feature !**

Devel branch has an improved netlist support. Maybe an **enable flag** on element, a kind of "**group**" feature and a **visitor** API would be nice ???

**Actually**, thanks to Python,

**1)** you can monkey patch Netlist with such code :

```
  def remove_element(self, element):
            del self._elements[element.name]
            self._dirty = True
```

**2)** or iterate on` _elements` using `element_iterator` and copy some elements to a new circuit.

**3)** or patch` Netlist.__str__` and Element for enable flag.

**Comment by FabriceSalvaire** [comment]
fixed in 6c48fb1
